interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/FragmentComponentDefinitionExtractor.scala

```diff
@@ -76,25 +76,34 @@ class FragmentComponentDefinitionExtractor(componentConfig: String => Option[Sin
       .mapWritten(_.map(data => FragmentParamClassLoadError(data.fieldName, data.refClazzName, nodeId.id)))
   }
 
-  private def toParameter(componentConfig: SingleComponentConfig)(p: FragmentParameter): Writer[List[FragmentParamClassLoadErrorData], Parameter] = {
-    val paramName = p.name
-    p.typ.toRuntimeClass(classLoader).map(Typed(_))
+  private def toParameter(componentConfig: SingleComponentConfig)(fragmentParameter: FragmentParameter): Writer[List[FragmentParamClassLoadErrorData], Parameter] = {
+    fragmentParameter.typ.toRuntimeClass(classLoader).map(Typed(_))
       .map(Writer.value[List[FragmentParamClassLoadErrorData], TypingResult])
       .getOrElse(Writer
         .value[List[FragmentParamClassLoadErrorData], TypingResult](Unknown)
-        .tell(List(FragmentParamClassLoadErrorData(paramName, p.typ.refClazzName)))
-      ).map(toParameter(componentConfig, paramName, _))
+        .tell(List(FragmentParamClassLoadErrorData(fragmentParameter.name, fragmentParameter.typ.refClazzName)))
+      ).map(toParameter(componentConfig, _, fragmentParameter))
   }
 
-  private def toParameter(componentConfig: SingleComponentConfig, paramName: String, typ: typing.TypingResult) = {
-    val config = componentConfig.params.flatMap(_.get(paramName)).getOrElse(ParameterConfig.empty)
+  private def toParameter(componentConfig: SingleComponentConfig, typ: typing.TypingResult, fragmentParameter: FragmentParameter) = {
```

Comment:
What about validations that the fixedValues and the defaultValue have the same type as the declared type?